### DISCUSSION TITLE
Freelancers

### DISCUSSION THREAD

**Discussion by tjbeevor** [langgenius/dify#5311] [Help]
Hey, this technology is really cool - and 'new'. I'm wondering if there are any folks in the community looking to 'freelance' and work in a distributed team to work on a whole host of business problems and help train some other developers while they're at it. Trying to learn it from scratch is tricky. Just wondering how people find skilled resources?

**Comment by perzeuss** [comment]
A great start is the discord community. We already have some experienced builders in the community. Chatting about ideas and helping each other is the best way to learn how to work with Dify.
